src/data/hullmods/hull_mods.csv

```diff
@@ -77,16 +77,16 @@ relatively frequent,vayra_ghost_ship_activity_4,,,Vayra,,,,,TRUE,TRUE,0,0,0,0,da
 
-Supply cost to recover from deployment reduced by %s.",Reduces weapon turn rates and durability.,graphics/hullmods/destroyed_weapon_mounts.png
+Supply cost to recover from deployment reduced by %s.",Reduces weapon turn rates and durability and increases their malfunction chance.,graphics/hullmods/destroyed_weapon_mounts.png
 Inoperative Automated Systems,vayra_damaged_automation,,,Vayra,"dmod, damage, civ",,,,TRUE,,0,0,0,0,data.scripts.hullmods.VayraDamagedAutomation,"Due to flaws introduced during manufacturing or possibly battle damage, automated systems aboard the ship that would normally assist the crew are almost entirely inoperable. Maximum combat readiness reduced by %s, and minimum crew required increased by %s.
 
 Supply cost to recover from deployment reduced by %s.",Increases minimum crew required and reduces maximum CR.,graphics/hullmods/vayra_very_not_automated_ship.png
 Performance Irregularities,vayra_damaged_everything,,,Vayra,"dmod, damage",,,,TRUE,,0,0,0,0,data.scripts.hullmods.VayraDamagedEverything,"This ship suffers from widespread malfunctions and minor glitches, all of which combine to produce a marked negative effect on its overall combat performance. Maximum combat readiness reduced by %s, and has a %s chance of suffering malfunctions during battle.
 
 Supply cost to recover from deployment reduced by %s.",Reduces maximum CR and has a small chance of causing malfunctions.,graphics/hullmods/vayra_damaged_everything.png
-Malformed Fighter Weapons,vayra_damaged_fighterweapons,,,Vayra,"dmod, fighterBayDamage",,,,TRUE,,0,0,0,0,data.scripts.hullmods.VayraDamagedFighterWeapons,"This ship's fighter manufactory is damaged or glitched in a particular manner that causes no ill effect to the general performance of chassis it produces, but dramatically reduces the effectiveness of their weapons. Fighter damage reduced by %s, and accuracy reduced by %s.
+Malformed Fighter Weapons,vayra_damaged_fighterweapons,,,Vayra,"dmod, fighterBayDamage",,,,TRUE,,0,0,0,0,data.scripts.hullmods.VayraDamagedFighterWeapons,"This ship's fighter manufactory is damaged or glitched in a particular manner that causes no ill effect to the general performance of chassis it produces, but dramatically reduces the effectiveness of their weapons. Fighter damage reduced by %s, Fighter accuracy reduced by %s and Fighter recoil increased by %s.
 
 Supply cost to recover from deployment reduced by %s.",Reduces damage dealt by and accuracy of launched fighters and bombers.,graphics/hullmods/vayra_damaged_fighterweapons.png
```

Comment:
Reduces damage dealt by and accuracy and increases recoil of launched fighters and bombers

Reply:
done. resolving.